modules/administration-guide/partials/proc_verification-custom-registries.adoc

```diff
@@ -0,0 +1,73 @@
+// deploying-the-registries
+
+[id="verification-custom_registries-_{context}"]
+
+.Verification steps
+
+. To verify that the new plug-in is correctly published to the plug-in registry, make a request to the registry path `/v3/plugins/index.json` (or `/devfiles/index.json` for the devfile registry).
++
+[subs="+quotes,+attributes"]
+----
+$ URL=$({orch-cli}  get -o 'custom-columns=URL:.spec.rules[0].host' \
+  -l app=che-plugin-registry {platform-routing} --no-headers)
```

Comment:
routes do not have `.spec.rules[0].host` but `.spec.host`

Reply:
and `-l app=che-plugin-registry` doesn't select anything: the route is not labelled. It would be better to use the name, at least the command would work.

Reply:
My fault. I'll fix it!

Reply:
> and -l app=che-plugin-registry doesn't select anything: the route is not labelled. It would be better to use the name, at least the command would work.

Good catch! The issue is that helm chart and openshift template uses different objects names. 
I'll take a look at the best option how to align them

Reply:
@tolusha Do you have any preference here?
Helm charts use `che-plugin-registry` while openshift template and che operator use plugin registry https://github.com/eclipse/che-operator/blob/cc937352745b8202a1f69d99b483b5ae0cf0374e/pkg/deploy/const.go#L4

Personally, I prefer che-plugin-registry a bit more because I would say plugin-registry is too general, but it may be OK if che leaves in a dedicated namespace which should be the case.

Reply:
Anatoliy brought a good point: During Click Party Deploy team discovered that custom registries docs are not working at all, apart from minor commands mistakes Mario found, there is also Http + Https mixing.
So, I've introduced one more variable that solves che-plugin-registry/plugin registry issue. There is an issue to review and update this article https://issues.redhat.com/browse/RHDEVDOCS-1762

Reply:
@sleshchenko 
I prefer to use labels that are used for operator and helm installer:
`app: che`, `component: plugin-registry`

Reply:
The PR on Che side is created https://github.com/eclipse/che/pull/18655
And docs are updated to use labels.